nodejs/week1/week1homework/src/bakckend/data/meals.json

```diff
@@ -0,0 +1,28 @@
+[
+    {
+      "id": 1,
+      "title": "Indian food in the summer",
+      "maxNumberOfGuests": 5,
+      "description": "A nice night out eating delicious indian food",
+      "createdAt": "2019/12/7 14:34",
+      "price": 67
+    },
+    {
+        "id": 2,
+        "title": "XL burger",
+        "maxNumberOfGuests": 12,
+        "description": "softice American burger",
+        "createdAt": "2020/03/10 14:34",
+        "price": 90
+      },
+      {
+        "id": 3,
+        "title": "buffe",
+        "maxNumberOfGuests": 15,
+        "description": "buffe and ful salad menue",
+        "createdAt": "2020/02/7 14:34",
+        "price": 120
```

Comment:
Fix formatting. This file is using both 2 spaces and 4 spaces for indentation.

Tip: Set up your text editor with something like prettier. It will automatically format your code every time you save the file so you don't need to even think about it.

For example, VSCode has this plugin https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode